src/Vanilla/Cli/Command/NodeCommandBase.php

```diff
@@ -10,16 +10,27 @@
 use \Garden\Cli\Args;
 use \Garden\Cli\Cli;
 use \Vanilla\Cli\CliUtil;
+use \Exception;
 
 /**
  * Class NodeCommandBase.
  */
 abstract class NodeCommandBase extends Command {
 
-    // We require node 8 because it is the latest LTS with support for async/await
+    /** @var string We require node 8 because it is the latest LTS with support for async/await */
     const MINIMUM_NODE_VERSION = '8.0.0';
 
-    protected $nodeVersion;
+    /** @var array */
+    public $dependencyDirectories = [];
```

Comment:
No public properties. Go with private or protected if you know you will use it in a subclass.